config/sounds-web/all.js

```diff
@@ -1,25 +1,28 @@
 const baseUrl = process.env.A11Y_SOUNDS_WEB_BASE_URL || 'https://www.bbc.co.uk';
 
+const { options } = require('./common');
+
 module.exports = {
-  options: {
-    hide: ['orb', 'bbccookies-prompt', 'smphtml5iframebbcMediaPlayer', 'smphtml5iframesmp-wrapper', 'edr_l_first', 'bbcprivacy-prompt', 'id4-cta-', 'msi-modal', 'p_audioui_'],
-    skip: []
-  },
+  options,
   baseUrl,
   paths: [
     '/sounds',
```

Comment:
Think you could update this to pull in the paths from the other config files to save defining the paths in two places.

Reply:
Done, Please have a look.